src/components/shared/DropDown/index.tsx

```diff
@@ -1,48 +1,61 @@
 import React, { useState, useRef, PropsWithChildren } from 'react';
 
+import UpArrow from '@src/assets/images/UpArrow.svg';
+import bluecircle from '@src/assets/images/bluecircle.svg';
+import greencircle from '@src/assets/images/greencircle.svg';
 import { ListType } from '../../../constants/dropDownDataList';
 import { useOutsideClick } from '@src/hooks/useOutsideClick';
 import * as S from './styled';
 
 type Props = {
+	isAd?: boolean;
+	adColor?: string;
+	isBig?: boolean;
 	list: ListType[];
 } & PropsWithChildren;
 
-const DropDown = (props: Props) => {
+const DropDown = ({ ...props }: Props) => {
 	const [isOpen, setIsOpen] = useState(false);
 	const [clickedData, setClickedData] = useState(props.list[0].value);
 	const clickRef = useRef() as React.MutableRefObject<HTMLDivElement>;
 
 	useOutsideClick(clickRef, setIsOpen);
 
+	const handleClick = (value: string): React.MouseEventHandler<HTMLLIElement> | undefined | void => {
+		setIsOpen(!isOpen);
+		setClickedData(value);
+	};
+
 	return (
-		<S.Container ref={clickRef}>
-			<div>
-				<button
-					onClick={() => {
-						setIsOpen(!isOpen);
-					}}
-				>
-					{clickedData}
-				</button>
-			</div>
+		<S.Container isBig={props.isBig} ref={clickRef}>
+			<S.SeletedData
+				isBig={props.isBig}
+				onClick={() => {
+					setIsOpen(!isOpen);
+				}}
+			>
+				{props.isAd && (props.adColor === 'blue' ? <img src={bluecircle} /> : <img src={greencircle} />)}
```

Comment:
이미지 소스에 대해서만 분기를 하면 되지 않을까요 ??
그리고, 이미지 태그에 대해서는 적어도 `alt` 속성을 항상 지정하는 게 웹 접근성 측면에서 좋습니다.

왜 인지는 검색해보시면 이유 정리해놓은 글을 찾아보실 수 있을 겁니다 !

```suggestion
				<img src={props.isAd && props.adColor === 'blue' ?  bluecircle : greencircle} alt="" /> 
```